packages/ui/src/forum/components/PostList/PostListItem.tsx

```diff
@@ -60,6 +65,16 @@ export const PostListItem = forwardRef<HTMLDivElement, PostProps>(({ post, isSel
           <Button square disabled={isPreview}>
             <ReplyIcon />
           </Button>
+          {isOwn && (
+            <ContextMenu
```

Comment:
Could you check (and add if missing) handling for a smaller context item? I think it should allow passing `size` prop similar to button sizing.